### ISSUE TITLE
About Encryptr menu should have link to the homepage + description

### ISSUE THREAD

**Issue by daviddahl** [SpiderOak/Encryptr#85]
I think it would help understanding if the "About" dialog had a link back to the Encryptr home page and perhaps a sentence describing the app in layman's terms, something like:

"Encryptr safely stores private data, passwords and credit card numbers in the cloud"

Also, a link to Crypton.io might be nice with a tagline like "Privacy by Crypton" :)

**Comment by helveticade** [comment]
We'll heavily revise the about screen for 3.0
